docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb

```diff
@@ -182,18 +171,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 8,
    "metadata": {},
    "outputs": [],
    "source": [
     "params = {\n",
     "    \"module__n_layers\": 2,\n",
     "    \"module__n_w_bits\": 4,\n",
     "    \"module__n_a_bits\": 4,\n",
-    "    \"module__n_accum_bits\": 15,\n",
```

Comment:
no more `module__n_accum_bits` ??

Reply:
PoT + Rounding feature handles the accumulator by itself. Nothing to do yes.

Reply:
I see thanks !